### ISSUE TITLE
replace `\\( ... \\)` with `$ ... $`

### ISSUE THREAD

**Issue by VladimirAlexiev** [qudt/qudt-public-repo#983]
Latex allows inline math with the two ways in the title of this issue. For uniformity, I propose to replace the first way with the second, which is used a lot more often.
Here are the occurrences:
```
10 matches for "\\(" in buffer: units.ttl
   2504:  qudt:expression "\\(Btu/(hr-ft^{2})\\)"^^qudt:LatexString ;
   2814:  qudt:expression "\\(Btu/(s-ft^{2})\\)"^^qudt:LatexString ;
   9293:  qudt:latexSymbol "\\(ft \\cdot lbf/(ft^2 \\cdot s)\\)"^^qudt:LatexString ;
  14654:  qudt:latexSymbol "\\(J/(kg \\cdot K)\\)"^^qudt:LatexString ;
  22894:  A bar is equal to $100,000\\ pascals\\(Pa)$, approximately equal to the atmospheric pressure on Earth at sea level.
  27079:  qudt:expression "\\(\\frac{\\mu m}{min}\\)"^^qudt:LatexString ;
  32231:  qudt:latexSymbol "\\(N \\cdot s \\cdot m^{-3}\\)"^^qudt:LatexString ;
  36749:  qudt:latexSymbol "\\(m^{-1} \\cdot T^{-1}\\)"^^qudt:LatexString ;
  41922:  qudt:expression "\\(therm-eec\\)"^^qudt:LatexString ;
  44582:  qudt:latexSymbol "\\(W \\cdot m^{-1} \\cdot K^{-1}\\)"^^qudt:LatexString ;
```
(line 22894 is https://github.com/qudt/qudt-public-repo/issues/982)

**Comment by steveraysteveray** [comment]
Are you sure you are using the latest version of main? I think all these changes have already been made.

**Comment by VladimirAlexiev** [comment]
Thanks! The report was against  2.1.41, it's fixed in  2.1.43
